src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.ComponentModel;
```

Comment:
Do you know what is bringing this in?

Reply:
Yes, the attribute         `[EditorBrowsable(EditorBrowsableState.Never)]`
We use it elsewhere in the library as well.